packages/ui/src/ui/pages/system/Masters/SwitchLeader.tsx

```diff
@@ -23,42 +23,61 @@ type SwitchLeaderPromtProps = {
-                <>
-                    <div className={b('address')}>
-                        <span>Old leader:</span>
-                        <span>
-                            {props.currentLeaderAddress}
-                            <ClipboardButton
-                                view="flat-secondary"
-                                text={props.currentLeaderAddress}
-                            />
-                        </span>
-                    </div>
-                    <div className={b('address')}>
-                        <span>New leader:</span>
-                        <span>
-                            {props.newLeaderAddress}
-                            <ClipboardButton view="flat-secondary" text={props.newLeaderAddress} />
-                        </span>
-                    </div>
-                </>
-            }
-            confirmText="Switch"
-            onCancel={props.cancel}
-            onConfirm={props.confirm}
-            onOutsideClick={props.cancel}
+        <YTDFDialog
             visible={props.visible}
-            loading={props.loading}
+            headerProps={{
+                title: 'Are you sure that you want to switch the cell leader?',
+            }}
+            fields={[
+                {
+                    type: 'block',
+                    name: 'text',
+                    extras: {
+                        children: (
+                            <>
+                                <div className={b('address')}>
+                                    <span>Old leader:</span>
+                                    <span>
+                                        {props.currentLeaderAddress}
+                                        <ClipboardButton
+                                            view="flat-secondary"
+                                            text={props.currentLeaderAddress}
+                                        />
+                                    </span>
+                                </div>
+                                <div className={b('address')}>
+                                    <span>New leader:</span>
+                                    <span>
+                                        {props.newLeaderAddress}
+                                        <ClipboardButton
+                                            view="flat-secondary"
+                                            text={props.newLeaderAddress}
+                                        />
+                                    </span>
+                                </div>
+                            </>
+                        ),
+                    },
+                },
+            ]}
+            footerProps={{
+                textApply: 'Switch',
+            }}
+            onAdd={(_form) => {
+                props.confirm();
+
+                return Promise.resolve();
+            }}
+            onClose={props.cancel}
+            isApplyDisabled={isApplyDisabled}
```

Comment:
Use `pristineSubmittable={true}` instead of `isApplyDisabled`. isApplyDisabled is required when you need to check values of form.